code/modules/clothing/factions/newhaiphong.dm

```diff
@@ -17,8 +17,8 @@
 	item_state = "aodaicrop"
 
 /obj/item/clothing/head/nonla
-	name = "non la"
-	desc = "A conical straw hat enjoyed particularly by residents of New Hai Phong, to protect the head from sweltering suns and heavy rains."
+	name = "conical hat"
```

Comment:
keep straw hat in the name or just name it a non la, "conical hat" makes me think of the idiot cone hat you'd see in cartoons